package.json

```diff
@@ -2,22 +2,34 @@
   "name": "funcadelic",
   "version": "0.3.0",
   "description": "Functional Fundamentals are Fun!",
-  "main": "nodule.js",
-  "module": "src/funcadelic.js",
+  "main": "dist/funcadelic.cjs.js",
+  "module": "dist/funcadelic.esm.js",
+  "browser": "dist/funcadelic.umd.js",
   "repository": "https://github.com/cowboyd/funcadelic.js.git",
   "author": "Charles Lowell <cowboyd@frontside.io>",
   "license": "MIT",
   "private": false,
   "scripts": {
-    "test": "mocha tests/index.js"
+    "build": "rollup -c",
```

Comment:
I'm a fan of using short options from the CLI, but full options in scripts. That way it's optimized for readibility. Let's expand this to `rollup --config`

Reply:
ok